ED/src/dynamics/photosyn_driv.f90

```diff
@@ -14,7 +14,8 @@ subroutine canopy_photosynthesis(csite,cmet,mzg,ipa,ntext_soil
    use pft_coms       , only : water_conductance  & ! intent(in)
                              , include_pft        & ! intent(in)
                              , vm0                & ! intent(in)
-                             , leaf_turnover_rate ! ! intent(in)
+                             , leaf_turnover_rate & ! intent(in)
+                             , leaf_psi_tlp       ! ! intent(in)
```

Comment:
This will probably gibe a warning of unused variable

Reply:
Hi, Manfredo

Thanks for your review. This leaf_psi_tlp was used for test purposed and is supposed to be used in my next pull request. I will probably update this pull request with my other new updates (drought phenology, leaf water potential modified stomatal conductance). And perform another round of test suites.

As for M34, I have asked Marcos, he said he encountered this problem before and changes the RK4 tolerance to avoid the RK4 Underflow error. But I'm not sure which modification leads to the error.

Reply:
Ok. I think it's definitely no prob to update with the next PR. Also I think once this is done it would be great to have part of the test to cover your new features.
To be honest I was also planning to add PFT 17 to the test for at least the GYF site but I didn't have time yet.

Reply:
Yeah, I'm planning to add Costa Rican sites as additional test suites. Maybe we can coordinate and add two test sites into the suites.

Xiangtao

Reply:
Yes, by the way, I think the Test suite itself also needs some upgrade. I rewrote the script (https://github.com/manfredo89/ED2/blob/redux/EDTS/exec_test_carver_serial.sh and https://github.com/manfredo89/ED2/blob/redux/EDTS/dev_test.m) that creates the qsub jobs so that we need only one ED2IN template per site (so as to avoid the risk of having different parameters for DBUG, TEST and MAIN). Hopefully I'll find time to push it to the main branch.